Lake/CLI/Main.lean

```diff
@@ -235,7 +241,7 @@ protected def list : CliM PUnit := do
   processOptions lakeOption
   let config ← mkLoadConfig (← getThe LakeOptions)
   noArgsRem do
-    let ws ← loadWorkspace config
+    let ws ← loadWorkspace config |>.run config.verbosity
```

Comment:
Most of these `|>.run config.verbosity` seem like they could be avoiding by just defining a `LogIO -> Cli(State)M` instance that uses to the `verbosity` option to lift `LogIO`.

Reply:
I've done this in most cases. Note that you have to be careful with the `MonadLift LogIO MainM` instance: for example the `loadWorkspace config |>.run config.verbosity` also typechecks if you leave the ` |>.run config.verbosity` part off but it is incorrect since the `MainM` monad doesn't have access to the verbosity state.